test/CMakeLists.txt

```diff
@@ -24,12 +24,24 @@ enable_testing()
 # Copy test files to output directory
 file(COPY resources DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 
-# Copy plugins to output plugin directory
-ADD_CUSTOM_COMMAND(TARGET lightdb_test
-        POST_BUILD
-        COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/plugins
-        COMMAND ${CMAKE_COMMAND} -E copy ${PROJECT_SOURCE_DIR}/plugins/* ${CMAKE_CURRENT_BINARY_DIR}/plugins
+# Build plugins used by tests
+include(ExternalProject)
+file(GLOB LIGHTDB_PLUGIN_DIRS ${CMAKE_SOURCE_DIR}/plugins/*)

```

Comment:
Is there no flag on `file` that would let you skip the `IS_DIRECTORY` check?

Reply:
I did look for that, but unfortunately didn't find any options skip non-directories.